### ISSUE TITLE
Connector build report: debug and fix

### ISSUE THREAD

**Issue by davydov-d** [airbytehq/airbyte#15604]
The connector build report posts out of date build fails from time to time. 
For instance, today's report contains same two failed builds it contained yesterday:

![image](https://user-images.githubusercontent.com/7703835/184368515-d455ed77-1f31-44c7-9f01-6f5fee886f29.png)

https://github.com/airbytehq/airbyte/actions/runs/2839336038
https://github.com/airbytehq/airbyte/actions/runs/2839339849

Both failed on 11.08.22. Both posted on 11.08.22 and 12.08.22, although there are two green builds which ran after those in the 
reports: 
https://github.com/airbytehq/airbyte/actions/runs/2845471624
https://github.com/airbytehq/airbyte/actions/runs/2840286416

In other words, these two should not have been included in the latest report. This distracts from actual fails. Need to figure out the reason and fix it
